go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/zonedb/zonedb
 // Minimum supported Go version
 // (current version -2, except when security fixes are only backported to the current version -1)
 // Note: this should use the latest dot release in CI, once GitHub has added it to "actions/go-versions"
-go 1.23.0
```

Comment:
should the comment above be updated?

Reply:
It still applies. I added a comment about `toolchain`.

Reply:
It might be clearer to say `GHA will use...` since that's what this means.